.gitignore

```diff
@@ -11,4 +11,5 @@ configs/local/
 *.log
 **/*.log
 **/credentials/
+.idea
```

Comment:
Do you need to include the .iml file in here too?

Reply:
git rm --cached .idea
 on PR #110